### PR TITLE
Jsfunc

### PR DESCRIPTION
**Pull request by gotchoices**
This is my latest.  Everything was working fine, but I integrated some input from Jim Schlight (a node-addon developer) about better handling of napi_env objects.

### CODE REVIEW

File: src/tclbinding.cpp [hunk 1]
```diff
@@ -96,6 +90,7 @@ Napi::Value TclBinding::cmdSync( const Napi::CallbackInfo& info ) {
 	}
 
 	std::string cmd = info[0].As<Napi::String>().Utf8Value();
+//printf("I1:%p c:%s\n", this->_interp, cmd.c_str());
```
Comment (uditha-atukorala) [inline src/tclbinding.cpp:93]: Assume this was left over from debugging?

File: README.md [hunk 2]
```diff
@@ -159,6 +165,31 @@ tcl.queue( 'incr x', function ( err, result ) {
...
+
+| Name     | Type   | Description               |
+|----------|--------|---------------------------|
+| cmd      | String | New Tcl command to create |
+| callback | [Callback](http://nukedzn.github.io/node-tcl/docs/Tcl.html#~cmdCallback) | Callback method to handle the command |
+
+#### Example
+
+``` js
+tcl.jsFunc( 'myNewCmd', function ( arg1, arg2 ) {
```
Comment (uditha-atukorala) [inline README.md:185]: Could we use `tcl.proc( cmd, callback )` signature for this? It's what we've discussed in #8.

**Review by uditha-atukorala — CHANGES_REQUESTED** [review]
Overall this looks good! Thanks for taking time to do the changes 🙇‍♂️. 

I've requested a couple of changes to the PR, mainly to align with what we've discussed before on how to expose JS functions within Tcl.

File: src/tclbinding.cpp (hunk 1 above)
Comment (gotchoices) [reply src/tclbinding.cpp:93]: Yes, sometimes I leave commented debug commands at key points where it has been handy to examine intermediate results.  I can take it out if you like.

File: README.md (hunk 2 above)
Comment (gotchoices) [reply README.md:185]: Assuming you just (and only) mean to rename the function name.  That's no problem.

**Comment by gotchoices** [comment]
I have made those changes and pushed to my repo.  Do I need to take other action (like a fresh PR)?  Or do you automatically have this latest commit?

**Review by uditha-atukorala — APPROVED** [review]

**Pull request merged**